schedstat.go

```diff
@@ -112,17 +125,17 @@ func parseProcSchedstat(contents string) (stats ProcSchedstat, err error) {
 }
 
 func (stat *SchedstatCPU) RunningSeconds() float64 {
-	return float64(stat.RunningJiffies) / userHZ
+	return float64(stat.RunningNanosec) / 1000000000
```

Comment:
Minor Go style nit. I would prefer to define `1000000000` as `const nanoSeconds = 1000000000`.

Reply:
even better, I could just use `time.Duration`. It's defined as `int64`, whereas the kernel defines the counters as `unsigned long long`. So after about 290 years of uptime, there will be an overflow problem. But that seems practically unlikely. Well before that, there will be issues with float precision, so I guess anyone wanting precise metrics with prometheus will need to reboot their servers every 100 years or so.

Reply:
No, we've run into issues using `time.Duration` with this library previously[0].

Actually, want I'd like to do is follow the same pattern. Have the function return `RunningNanoseconds` as the raw uint64, and leave it up to the user (node_exporter) to convert from uint64 nanoseconds into float64 seconds.

[0]: https://github.com/prometheus/procfs/pull/165